exporter/awsxrayexporter/awsxray.go

```diff
@@ -90,6 +90,7 @@ func NewTraceExporter(config configmodels.Exporter, logger *zap.Logger, cn connA
 				if localErr != nil && !config.(*Config).LocalMode {
 					err = wrapErrorIfBadRequest(&localErr) // not test mode, so record error
 				}
+				logger.Debug("localErr: " + localErr.Error())
```

Comment:
This needs to be inside a `localErr != nil` check. I think we can remove `!localMode` from that check too no need to avoid the error even in local mode.

Reply:
Sounds sweet! Let me do it!